src/random_test.cpp

```diff
@@ -2797,6 +2810,10 @@ bool execute_sql(const std::string &sql, Thd1 *thd) {
 
   return (res == 0 ? 1 : 0);
 }
+// Retrive the formed deque
+std::deque<std::string> get_recent_queries() {
```

Comment:
Seems like you duplicating the function?

Reply:
Removed as its already present in Thd Class